creational/object-pool.md

```diff
@@ -10,14 +10,14 @@ package pool
 
 type Pool chan *Object
 
-func New(total int) *Pool {
+func New(total int) Pool {
```

Comment:
This is not compatible with https://github.com/tmrts/go-patterns/pull/73/files#diff-e59192bbfdcbd3b94d0435f6a0dd5d04R32 because we can't read from a channel pointer without dereferencing it.